src/python/grpcio/grpc/_server.py

```diff
@@ -670,6 +670,10 @@ def _on_call_completed(state):
 
 def _serve(state):
     while True:
+        # We want to force the deletion of the last event         
```

Comment:
"last" easily misreads as "terminal", "final", and "ultimate" - can this be restated in terms of "current" and "next" or "previous" and "next"?